### PR TITLE
wic: Use custom kernel path if provided

### PR DESCRIPTION
**Pull request by billpittman**
If the custom kernel path is provided in options, then
use that path instead of the default path.

Signed-off-by: Bill Pittman <bill.pittman@ni.com>

### CODE REVIEW

File: scripts/wic [hunk 1]
```diff
@@ -159,6 +159,9 @@ def wic_create_subcommand(options, usage_str):
                            "(Use -e/--image-name to specify it)")
         native_sysroot = options.native_sysroot
 
+    if options.kernel_dir:
```
Comment (shruthi-ravi) [inline scripts/wic:162]: There seems to be an assumption above and below that only when `option.image_name` is NOT defined, kernel_dir is read from the options. I don't know why that is, but this line seems to be at odds with that assumption.. 
This line probably needs more comments to explain why we are breaking that assumption, if we decide to keep it.

**Review by amstewart — COMMENTED** [review]
Is this commit going to be submitted to OE-core upstream?

**Comment by billpittman** [comment]
Already submitted upstream.  As Shruthi mentioned though there seems to be an "assumption" or "oversight" of the use of this variable in the code, so I'm sort of expected some push back from upstream, but haven't seen it yet.

Upstream patch: https://lists.openembedded.org/g/openembedded-core/message/161756

**Comment by amstewart** [comment]
OK. In that case, could you replace this PR with a cherry-pick from upstream/master, once it has been accepted?

**Comment by billpittman** [comment]
> OK. In that case, could you replace this PR with a cherry-pick from upstream/master, once it has been accepted?

Sure, just want to keep the issue "broken" until the upstream PR is accepted?

**Comment by billpittman** [comment]
Here's the upstream commit: https://git.openembedded.org/openembedded-core/commit/scripts?id=1068102216a894c467f71f6046fdb37d5577545c

**Comment by amstewart** [comment]
Nice! One more thing. Could you replace the commit in the PR with a cherry-picked version of the commit from upstream/master, with `-x` specified, so that it adds the `Cherry-picked from: ...` trailer? It helps at rebase-time. :)

**Comment by billpittman** [comment]
Done - you should be good to go!

**Review by amstewart — APPROVED** [review]

**Pull request merged**